src/sys/windows/tcp.rs

```diff
@@ -397,6 +405,10 @@ impl StreamImp {
             self.inner.socket.write_overlapped(&buf[pos..], overlapped)
         };
         match err {
+            Ok(true) if me.instant_notify => {
+                me.write = State::Empty;
+                self.add_readiness(me, Ready::writable());
```

Comment:
Ok, seems to work.
Still feels slightly slower (might be worth to investigate it that's just a placebo or if doing
it like libuv/go is advantageous performance-wise.